packages/transformers/svg/src/dependencies.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import type {AST, MutableAsset} from '@parcel/types';
+import PostHTML from 'posthtml';
+
+// A list of all attributes that may produce a dependency
+// Based on https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute
+const ATTRS = {
+  href: ['a', 'use'],
+  'xlink:href': ['a', 'use'],
+};
+
+// Options to be passed to `addDependency` for certain tags + attributes
+const OPTIONS = {
+  a: {
+    href: {needsStableName: true},
+    'xlink:href': {needsStableName: true},
+  },
+};
+
+export default function collectDependencies(asset: MutableAsset, ast: AST) {
+  let isDirty = false;
+  const stylesheets = [];
+  PostHTML().walk.call(ast.program, node => {
+    if (typeof node === 'string' && node.startsWith('<?xml-stylesheet')) {
+      const [, href] = node.match(/(?<=(?:^|\s)href\s*=\s*")(.+?)(?=")/i) ?? [];
+
+      if (!href) {
+        return node;
+      }
+      stylesheets.push(href);
```

Comment:
This is interesting. In HTML, we keep separate `<link rel="stylesheet">` elements separate even after bundling. What was the reasoning for merging them here?

Reply:
External resources in an SVG won't load when using ` <img>` (but will with `<object>`), but was thinking about this and it's the wrong place to handle it: if anywhere the packager could recognise if the SVG is used in an `<img>` and inline resources.